assembl/static2/js/app/components/common/modifyPasswordForm.jsx

```diff
@@ -0,0 +1,122 @@
+// @flow
+import React from 'react';
+import { compose, graphql } from 'react-apollo';
+import { Translate, I18n } from 'react-redux-i18n';
+import { Button } from 'react-bootstrap';
+
+import { displayAlert } from '../../utils/utilityManager';
+import UpdateUserMutation from '../../graphql/mutations/updateUser.graphql';
+import FormControlWithLabel from './formControlWithLabel';
+
+type Props = {
+  id: string,
+  updateUser: Function,
+  successCallback: Function
+};
+
+type State = {
+  disabled: boolean,
+  oldPassword: string,
+  newPassword: string,
+  newPassword2: string
+};
+
+class ModifyPasswordForm extends React.Component<void, Props, State> {
+  props: Props;
+
+  state: State;
+
+  state = {
+    disabled: false,
+    oldPassword: '',
+    newPassword: '',
+    newPassword2: ''
+  };
+
+  handleOldPasswordChange = (e: SyntheticEvent) => {
+    if (e.target instanceof HTMLInputElement) {
+      const value = e.target.value;
+      const disabled = value.length === 0 || this.state.newPassword.length === 0 || this.state.newPassword2.length === 0;
```

Comment:
maybe use a function or method for this to avoid repetition / for maintainability

Reply:
This is different each time here.

Reply:
or you mean to create a function like isEmpty?